app/workers/central_mail/submit_central_form686c_job.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+require 'central_mail/service'
+require 'central_mail/datestamp_pdf'
+require 'pdf_info'
+
+module CentralMail
+  class SubmitCentralForm686cJob
+    include Sidekiq::Worker
+    include SentryLogging
+
+    FOREIGN_POSTALCODE = '00000'
+    FORM_ID = '686C-674'
+
+    sidekiq_options retry: false
+
+    class CentralMailResponseError < StandardError
+    end
+
+    def perform(saved_claim_id, vet_info, user_struct)
```

Comment:
@tfink419 Can you take a look at encrypting the params or storing the data in the DB? Kyle is correct, these objects both have PII in them.

Reply:
I've just added encryption to the 4 jobs associated with this ticket

Reply:
I decided since that code takes this PR past 500 LoC as well as just muddles the overall goal of the PR, I've moved the changes to https://github.com/department-of-veterans-affairs/vets-api/pull/13916 . The PR will likely have merge conflicts anyways with another PR of mine in which I heavily modified some of the spec files as well

Reply:
resolving issue as we are fixing in a separate PR